src/components/benchmark/VisualizeBenchmarks.css

```diff
@@ -0,0 +1,11 @@
+
+.benchmark-value {
+  margin: 5px;
+  padding: 6px 24px;
+  border-radius: 10px;
+}
+
+/* TODO: does this have any effect? */
```

Comment:
Not sure

Reply:
Yea, for some reason the css didn't get picked up for the table. Had to specify the style explicitly in the component

Reply:
We can fix that later if needed.